tests/spread/plugins/v1/rust/cargo-lock/task.yaml

```diff
@@ -1,47 +0,0 @@
-summary: Build and run a basic rust snap with Cargo.lock
```

Comment:
Deleted?

Reply:
yes, we implicitly run these tests with the other tests that already have a lock.

Reply:
Agreed to add this back for v1